PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Binary/File/Builder/V1/MegFileBuilder.cs

```diff
@@ -6,13 +9,14 @@
 using System.Text;
 using PG.Commons.Binary.File.Builder;
 using PG.Commons.Util;
-using PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition;
+using PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition.V1;
```

Comment:
i don't really like the V1 V2 for namespaces. If the old version is deprecated let it die. Community is small enough to allow these kind of breaking changes.

Reply:
Already discussed on discord:
> I used that in that case because there are four versions of MEG archives. So technically support for the other versions (8bit Armies, Gery Goo, and C&C Remastered) isn't entirely out there, there is a fork of my older MEG library where a guy is implementing the C&C version for example.
So having all versions available in parallel is in this case required. The defining API itself, the IMegProcessService is not immutable.